src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -95,6 +96,38 @@ internal async Task ProcessHttpInAndOutInvocationRequest(ScriptInvocationContext
             }
         }
 
+        private async Task<HttpResponseMessage> SendInvocationRequestAsync(ScriptInvocationContext scriptInvocationContext, HttpRequestMessage httpRequestMessage)
+        {
+            // Only log Request / Response when running locally
+            if (_environment.IsCoreTools())
```

Comment:
>_environment.IsCoreTools() [](start = 16, length = 26)

I haven't double checked but is there an IOptions way to check if coreTools or not ?

Reply:
We do rely on `__environment`. Let me know if you have more concerns.